### DISCUSSION TITLE
in prep for week 15: third project iteration!

### DISCUSSION THREAD

**Discussion by benmiller314** [benmiller314/dsam2023fall#16] [Show and tell]
Hi, everyone! You can **use this space to post your maximum-_10_-minute videos**, along with a link to your most current [public-facing project deliverable](https://benmiller314.github.io/dsam2023fall/projects#palimpsest-of-public-iteration); we'll then be able to use the discussion forum to [write replies](https://benmiller314.github.io/dsam2023fall/projects#presentations-and-peer-review), too.

<div class="alert alert-success">To upload a video, just drag and drop the file onto your "Write" tab. If the file is too large for the discussion forum, consider hosting it elsewhere (OneDrive, for instance) and <a href="https://www.technology.pitt.edu/security/onedrive-security-guide#:~:text=Manage%20Sharing%20Permissions.">setting permissions</a> so we can access it via hyperlink.</div>

Just so you know (and I hope this puts you further at ease), I will *not* be requiring that you play your video in class while you listen to your own recorded voice and squirm. Instead, because you will likely have to practice to get your presentation to fit in 10 minutes, you'll be well-rehearsed to give your presentation live to the class, leaving us plenty of time to have a conversation about your work. (You're also welcome to just play the video if you prefer.)

See the [end of week 14's lesson plan](https://benmiller314.github.io/dsam2023fall/plans/week-14#hw) for more detailed guidance, but by the end of your presentation **I invite you to reflect on your process and tools**: given the problem/puzzle/question you started with, and the sources you brought to bear in response, how did the affordances and restrictions of your tools shape your direction? In addition to the claims or findings you feel ready to present, what were the frustrations and celebrations? What does that suggest to you for future iterations of this project?

If you're having trouble doing the above, you can always (instead) talk about why you're having trouble doing the above. But try to lean into the almosts, the felt senses, the murky places, if you can. Trying can sometimes surprise you.

**Comment by Khushboobhutani** [comment]
Link for the final project: https://scalar.usc.edu/works/dsam-project-revised

Video for the final presentation: 
https://github.com/benmiller314/dsam2023fall/assets/143055269/e0fffb73-26e0-4334-a0b4-b51d6d05daee

**Comment by chloe-cmu** [comment]
[DSAM Presentation 2023-12-04.zip](https://github.com/benmiller314/dsam2023fall/files/13540497/DSAM.Presentation.2023-12-04.zip)
[Food-Additives-Full-List 2023-12.csv](https://github.com/benmiller314/dsam2023fall/files/13540500/Food-Additives-Full-List.2023-12.csv)

**Comment by Rezaetmd** [comment]
Link to the final project on Scalar: https://scalar.usc.edu/works/filmfarsi/index
Link to the video:  https://www.dropbox.com/scl/fi/jxbcz0sw05fjm4x9hy2ue/Reza-DSAM-Project.MP4?rlkey=jhejx7ixi5a78cyea8kxf1mah&dl=0

**Comment by SidraNArshad** [comment]
Final Project: https://scalar.usc.edu/works/dsam-project---blm-activists-/tracing-external-narrative?path=the-ferguson-conspiracy

Video: https://clipchamp.com/watch/lpR8g3vxKu5

**Comment by amj169** [comment]
Resources for Iteration 3:

I just threw my video and Rfile in a zip file because it wouldn't support the knitted rfile. 

[DSAM Final Iteration Materials.zip](https://github.com/benmiller314/dsam2023fall/files/13541659/DSAM.Final.Iteration.Materials.zip)


Canva Presentation:
[DSAM Iteration 3 Update.pdf](https://github.com/benmiller314/dsam2023fall/files/13541613/DSAM.Iteration.3.Update.pdf)


Website Link: https://sites.google.com/d/1RljPXM-aoMi04aLCkWH1blNv2w9DQs9f/p/1kiBeMjB1CKYZ38avAUEljDdBl3hCtRGb/edit

**Comment by MustafaKandil** [comment]
Video link: https://pitt-my.sharepoint.com/:v:/r/personal/muk40_pitt_edu/Documents/video%20for%20DSAM/Mustafa%20Kandil.mov?csf=1&web=1&nav=eyJyZWZlcnJhbEluZm8iOnsicmVmZXJyYWxBcHAiOiJPbmVEcml2ZUZvckJ1c2luZXNzIiwicmVmZXJyYWxBcHBQbGF0Zm9ybSI6IldlYiIsInJlZmVycmFsTW9kZSI6InZpZXciLCJyZWZlcnJhbFZpZXciOiJNeUZpbGVzTGlua0RpcmVjdCJ9fQ&e=nfR0lp
Scalar link: https://scalar.usc.edu/works/1-85/index

Thank you!
